service/history/queryRegistry.go

```diff
@@ -192,6 +199,9 @@ func (r *queryRegistryImpl) getQuery(id string) (query, error) {
 	if q, ok := r.unblocked[id]; ok {
 		return q, nil
 	}
```

Comment:
Is this still needed since we are removing unblockQuery method?

Reply:
It is still needed. Even though we removed unblockQuery method we are not removing the ability to unblock a query. Basically now query state machine just allows posting a terminal state. One of those terminal states is unblocked query.

Reply:
It is still needed. Even though we removed unblockQuery method we are not removing the ability to unblock a query. Basically now query state machine just allows posting a terminal state. One of those terminal states is unblocked query.